src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -26,6 +26,30 @@
       <nuget:NullToBooleanConverter
         x:Key="NullToBooleanConverter" />
 
+      <!--Style for warning tooltip text-->
+      <Style TargetType="TextBlock" x:Key="VulnerabilitiesCountText">
+        <Setter Property="Text">
+          <Setter.Value>
+            <MultiBinding Converter="{StaticResource StringFormatConverter}">
+              <Binding Source="{x:Static nuget:Resources.Label_MultiplePackageVulnerableVersionToolTip}" />
+              <Binding Path="VulnerableVersionsString" />
+            </MultiBinding>
+          </Setter.Value>
+        </Setter>
+        <Style.Triggers>
+          <DataTrigger Binding="{Binding VulnerableVersions.Count}" Value="1">
+            <Setter Property="Text">
+              <Setter.Value>
+                <MultiBinding Converter="{StaticResource StringFormatConverter}">
+                  <Binding Source="{x:Static nuget:Resources.Label_PackageVulnerableToolTip}" />
+                  <Binding Path="VulnerabilityMaxSeverity" Converter="{StaticResource IntToVulnerabilitySeverityConverter}" />
+                </MultiBinding>
+              </Setter.Value>
```

Comment:
We shouldn't need the plural/singular form and data trigger now that we changed the text.